package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc",
     "start": "node -r tsconfig-paths/register -r ts-node/register ./dist/server.js",
     "typeorm": "ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js",
-    "test": "ENVIRONMENT=test jest",
+    "test": "ENVIRONMENT=test jest --runInBand",
```

Comment:
in my local environment, tests were constantly failing. Added this after reading the documentation from [typeorm-seeding](https://github.com/w3tecch/typeorm-seeding#-seeding-data-in-testing)